### ISSUE TITLE
Research performance concern related to binding `this` in the render method.

### ISSUE THREAD

**Issue by jeneve** [LookingForMe/lookingForFrontEnd#62]
from @adamji:

`I don't want to detract from any patterns that you've established here, just one point though.

using .bind(this) in the render() of your component is a known performance concern. Many are achieving the same binding by including the following in the constructor of the component:

constructor() {
    super();
    this.state = { jobs: {}, inputValue: "", activePage: 1, lastCall: "all" };

    this.onKeyUp = this.onKeyUp.bind(this);
}
This helps organize your component and provides a documentation-like structure. caveat this does affect React Hot-Loading

tl;dr : http://stackoverflow.com/a/31302493`
